cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -343,13 +350,24 @@ func createEndpointsLW(kubeClient *kclient.Client) *kcache.ListWatch {
 	return kcache.NewListWatchFromClient(kubeClient, "endpoints", kapi.NamespaceAll, kSelector.Everything())
 }
 
+func (ks *kube2sky) addServiceRecords(records *recordTree, svc *kapi.Service) {
```

Comment:
once we get rid of legacy names, this doesn't really need to be a separate function any more.